aws-samples/amazon-chime-sdk
# Allow to enable simulcast and priority video policies

**Description of changes:**
- Allow to select simulcast uplink and priority-based downlink policies
- Disable the simulcast in non-Chrome browser and priority-based on Firefox.
- Fix the issue where selecting webAudio or background blur clear out the form.

**Testing**

1. How did you test these changes?
- Join a meeting with 3 videos in Chrome with simulcast and priority-based downlink policies.
- Verify that simulcast is enabled 
- Verify that zoom in and out button only shown when priority-based downlink policy is enabled.

2. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it? Yes, the meeting demo.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
